src/AppInstallerCLICore/ExecutionContext.cpp

```diff
@@ -184,6 +184,7 @@ namespace AppInstaller::CLI::Execution
     void Context::SetTerminationHR(HRESULT hr)
     {
         m_terminationHR = hr;
+        m_isTerminated = FAILED(hr);

```

Comment:
```suggestion
        m_isTerminated = true;
```
Should stay consistent with `Terminate`.